en/faq/nginx-proxy.md

```diff
@@ -35,6 +35,80 @@ server {
 }
 ```
 
+# Using nginx with generated pages and a caching proxy as fallback
+
+If you have a high volume website with regularly changing content, you might want to benefit from Nuxt generate capabilities and [nginx caching](https://www.nginx.com/blog/nginx-caching-guide).
+
+Below is an exampe configuration which you will most likely need to adapt to your needs.
+
+```nginx
+proxy_cache_path  /PATH/TO/YOUR/CACHE levels=1:2 keys_zone=nuxt-cache:25m max_size=1g inactive=60m use_temp_path=off;
+
+map $sent_http_content_type $expires {
```

Comment:
Is this for serving generated version only?

Reply:
No, you could use the proxy_cache as well if you dont use generated pages. Actually the only part for the generated pages is setting the root and adding `try_files $uri/index.html` ;)

Reply:
I will change the description of the example to point that out, thanks

Reply:
Yes! But doesn't it override expires cache headers sent from Nuxt? (Those that sent from nuxt.config). I mean it's actually better idea to delegate such configurations to nginx layer. But users should be aware of that.